array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function showPeopleBelongIn(array, houseName) {
+  const filteredArray = array.filter((person) => person.house === houseName);
```

Comment:
Same here, I think a more descriptive name would be useful. It's filtered, but filtered how?

Maybe `peopleMatchingHouse`, or `peopleInHouse`, or ``peopleBelongingInHouse`?

Reply:
`filteredArray` changed to `peopleInHouse`.
Thanks.